server/database/dbConnection.js

```diff
@@ -0,0 +1,9 @@
+const { Pool } = require('pg');
+require('env2')('./config.env');
+
+let { ELECTION_DAY_DB_URL } = process.env;
+
+if (!ELECTION_DAY_DB_URL)
+  throw new Error('Enviroment variable TRADING_DB_URL must be set');
```

Comment:
This is confusing. Why is your error message saying `TRADING_DB_URL` when it seems like the name of the url is `ELECTION_DAY_DB_URL` ? Is there another env variable that should be added to the .env file?